--- 
Header header
 string port
 float64 humidity
 bool success